Updated Index.html

louiske65 opened the pull request:
This PR is for issue #1

louiske65 committed:
Update index.html

louiske65 committed:
Merge pull request #1 from louiske65/issue-1

Update index.html (issue-1)

dwcaraway commented on index.html:
```diff
@@ -36,17 +33,22 @@ <h3>Greater Miami Valley Computer Science Circle</h3>
+          <p>The Greater Miami Valley Computer Science Circle (GMVCSC), in its 3rd year of operation, is an organization dedicated to helping the students of the Miami Valley and beyond. Our Computer circle focuses on computational thinking and problem solving. We use Python/Java/HTML to teach concepts like Algorithm design, Web design, Mobile apps, Robotics, and Information Security. We currentl ...
+We meet from 10:00am-Noon the 1st and 3rd Saturday of the month during the school year (September-early May) in the Mathematical and Microbiological Sciences (MM) Building on the campus of Wright State University.
+
+If you would like to inquire about joining one of our groups, please contact either Mr. Ken Nelson (at ken.nelson@wright.edu) or Dr. Munsup Seoh (at munsup.seoh@wright.edu). Spaces are limited! If there is not space for your child at the time you email us, then you will be placed on our waiting list. Available spaces will be offered to those on our waiting list first, with priority given to those ...
```
You might want to wrap each paragraph in its own `<p>`...`</p>` tags. Within a paragraph tag, blank lines are ignored.

jmatos35 replied:
Louis, go ahead and make this change:

The Greater Miami Valley... is a new paragraph.
We meet from 10:00am... is another paragraph.
If you would like... is another paragraph.
If you or some you know... is another paragraph.
Lastly... is another paragraph.
Coordinators: is another paragraph.

dwcaraway commented on index.html:
```diff
@@ -36,17 +33,22 @@ <h3>Greater Miami Valley Computer Science Circle</h3>
+          <p>The Greater Miami Valley Computer Science Circle (GMVCSC), in its 3rd year of operation, is an organization dedicated to helping the students of the Miami Valley and beyond. Our Computer circle focuses on computational thinking and problem solving. We use Python/Java/HTML to teach concepts like Algorithm design, Web design, Mobile apps, Robotics, and Information Security. We currentl ...
+We meet from 10:00am-Noon the 1st and 3rd Saturday of the month during the school year (September-early May) in the Mathematical and Microbiological Sciences (MM) Building on the campus of Wright State University.
+
+If you would like to inquire about joining one of our groups, please contact either Mr. Ken Nelson (at ken.nelson@wright.edu) or Dr. Munsup Seoh (at munsup.seoh@wright.edu). Spaces are limited! If there is not space for your child at the time you email us, then you will be placed on our waiting list. Available spaces will be offered to those on our waiting list first, with priority given to those ...
```
For email addresses, you might want to use a hyperlink so that the user can click on the email address to open an email in their favorite mail clients. For example, you might rewrite `munsup.seoh@wright.edu` as `<a href="mailto:munsup.seoh@wright.edu" target="_top">munsup.seoh@wright.edu</a>`. This won't work in all browsers since I think it was abused a bit in the past but will draw attention to the addresses. Will start a conversation on issue thread about whether we want to use an alternate contact method instead of email addresses

jmatos35 replied:
Louis, go ahead and make this change suggested by Dave. Thanks.

dwcaraway reviewed (changes requested):
Looks great, thank you for volunteering to make these enhancements. I made a few small suggestions for enhancements, once they're in, should be good to merge.

louiske65 committed:
Made Enhancements To index.html

Wrapped each paragraph in <p>...</p> tags.
Added hyperlinks to email addresses.

Chameleonn commented:
I'm unsure of what I am supposed to be doing currently

On Wed, Apr 25, 2018 at 5:24 PM, louiske65 <notifications@github.com> wrote:

> I just made new changes to index.html
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/gmvcsc/gmvcsc.github.io/pull/2#issuecomment-384439120>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/Ak2B2AsuNCevvlLt53rFU8LwqoiGF-D-ks5tsOmLgaJpZM4Tincb>
> .
>

dwcaraway reviewed (approved):
Nice work :shipit:

dwcaraway commented:
@jmatos35 feel free to merge the PR when you're good with the changes

dwcaraway commented:
@Chameleonn students are signing up for (or are being assigned) jira issues, working them and pull-requesting them back. I'll make a couple that could be interesting.
